js/editTask.js

```diff
@@ -0,0 +1,36 @@
+import { EDIT, COMPLETE, DELETE_TODO } from "/js/constants.js";
+import handleSaveButtonClick from "/js/utility/SaveBtnAction.js";
+import handleDeleteButtonClick from "/js/utility/DltBtnAction.js";
+import handleCheckButtonClick from "/js/utility/ChkBtnAction.js";
+
+export default function editTask(element, taskList, id) {
```

Comment:
Can we also have a proper naming for these parameters as well?

Reply:
done!